content/projects/sql/_index.md

```diff
@@ -8,21 +8,14 @@ ready: True
 You should be familiar with:  
  {{% contentlink "topics/intro-to-relational-databases" %}}
 
-## Instructions
-
-Download MySQL and install on Ubuntu:
 
-```
-sudo apt update
-sudo apt upgrade
-wget http://repo.mysql.com/mysql-apt-config_0.8.10-1_all.deb
-sudo dpkg -i mysql-apt-config_0.8.10-1_all.deb
-sudo apt update
-sudo apt install mysql-server
 
-```
+## Installation:
+* We are using Postgres
+* Instead of installing Postgres on your computer, you can launch it with a docker composition follow the [INTRO TO DOCKER AND DOCKER-COMPOSE](https://umuzi-org.github.io/tech-department/topics/intro-to-docker/) for steps to run Postgres in the container.
```

Comment:
Use a {{contentlink}}. We use the contentlink shortcode when linking internally.

What happens if we decide to host this thing somewhere else? We'll have to search through everything for hardcoded urls. Use your Django lessons when thinking about links